backend/src/routers/suggestRouter.ts

```diff
@@ -0,0 +1,144 @@
+import { Router } from 'express';
+import { createSuggestion, getAllSuggestions} from '../services/suggestService.js';
+import { z } from 'zod';
+import { validateRequestBody } from 'zod-express-middleware';
+import sendApiValidationError from '../utils/sendApiValidationError.js';
+import { platformExists } from '../services/platformService.js';
+
+
+const suggestRouter = Router();
+
+/**
+ * @api {get} /api/v1/suggestions Request Suggestions
+ * @apiName GetSuggestions
+ * @apiGroup Suggestions
+ * @apiDescription Get all public suggestions
+ *
+ * @apiSuccess {Object[]} suggestions List of suggestions
+ *
+ * @apiSuccessExample Success-Response:
+ *  HTTP/1.1 200 OK
+ *  [
+ *   {
+ *    "name": "Game 1",
+ *    "description": "Game 1 description",
+ * 	  "platformName": "Steam",
+ *	  "releaseDate": "2023-04-13T00:00:00.000Z",
+ *	  "playtime": 60,
+ *	  "playerMin": 1,
+ *	  "playerMax": 5
+ *   }
+ * ]
+ */
+
+ suggestRouter.get('/', async (req, res) => {
+    const suggestion = await getAllSuggestions();
+    const formattedSuggestion = formatSuggestion(suggestion);
+    res.status(200).json(formattedSuggestion);
+});
+
+const addSuggestionSchema = z.object({
+    name: z.string().min(1).max(250),
+	description: z.string().min(1).max(2000),
+	platform: z.string().min(1),
+	releaseDate: z.string().datetime(), // ISO date string
+	playtime: z.number().int().min(1),
+	playerMin: z.number().int().min(1),
+	playerMax: z.number().int().min(1) //Maybe check that max > min?
+});
+
+
+
+
+
+/**
+ * @api {post} /api/v1/suggestions/add Add a suggestion
+ * @apiName AddSuggestion
+ * @apiGroup Suggestions
+ * @apiDescription Adds a suggestion to the service
+ *
+ * @apiBody {String} name Name of the game
+ * @apiBody {String} description Description of the game
+ * @apiBody {String} platform Platform the game is played on
+ * @apiBody {String} releaseDate Date the game was released
+ * @apiBody {Number} playtime Playtime of the game
+ * @apiBody {Number} playerMin PlayerMin of the game
+ * @apiBody {Number} playMax PlayerMax of the game
+ *
+ * @apiSuccess {String} message Message indicating success
+ *
+ * @apiSuccessExample Success-Response:
+ * HTTP/1.1 200 OK
+ * {
+ * 	"message": "Game added"
+ * }
+ *
+ * @apiUse ZodError
+ */
+suggestRouter.post(
+	'/add',
+	validateRequestBody(addSuggestionSchema),
+	async (req, res) => {
+		const body = req.body;
+
+		if (!(await platformExists(body.platform))) {
+			return sendApiValidationError(
+				res,
+				{
+					path: 'platform',
+					message: 'Platform does not exist'
+				},
+				'Body'
+			);
+		}
+        //Maybe change so release date can be in the future?
+		if (body.releaseDate > new Date().toISOString()) {
+			return sendApiValidationError(
+				res,
+				{
+					path: 'releaseDate',
+					message: 'The release date cannot be in the future'
+				},
+				'Body'
+			);
+		}
```

Comment:
I think, like your own comment says, this should be the case. Especially for suggested games, as those are even more likely to be later releases than games that someone currently has.
```suggestion
```